payments/schema/queries.py

```diff
@@ -141,12 +141,39 @@ def resolve_order_details(self, info, order_number):
         except Order.DoesNotExist as e:
             raise VenepaikkaGraphQLError(e)
 
-        order_type = OrderTypeEnum.UNKNOWN
         is_application_order = (
             hasattr(order, "lease")
             and getattr(order.lease, "application", None) is not None
         )
 
+        order_type = Query._get_order_type(order)
+
+        if isinstance(order.lease, BerthLease):
+            place_number = order.lease.berth.number
+            section_identifier = order.lease.berth.pier.identifier
+            area_name = order.lease.berth.pier.harbor.name
+        elif isinstance(order.lease, WinterStorageLease):
+            place_number = str(order.lease.place.number)
+            section_identifier = order.lease.place.winter_storage_section.identifier
+            area_name = order.lease.place.winter_storage_section.area.name
+        else:
+            place_number = None
+            section_identifier = None
+            area_name = None
+
+        return OrderDetailsType(
+            status=order.status,
+            order_type=order_type,
+            area=area_name,
+            section=section_identifier,
+            place=place_number,
+            is_application_order=is_application_order,
+        )
+
+    @staticmethod
+    def _get_order_type(order):
```

Comment:
Nitpick: can you move this to `payments/schema/utils.py`? I think it fits better there rather than as a method of the `Query` class 😇